src/ui/components/results/alternativeverticalscomponent.js

```diff
@@ -123,6 +123,17 @@ export default class AlternativeVerticalsComponent extends Component {
       params.set(StorageKeys.REFERRER_PAGE_URL, referrerPageUrl);
     }
 
+    let prefixes = this.componentManager.getComponentNamesForComponentTypes([
```

Comment:
This was going originally to be in a function in `urlutils`, until I got circular dependency errors because of a componentManager import. It seems we cannot access componentManager from urlutils and instead must rely on the class' property. I could add the function in urlutils to take `componentManager` as a parameter, but thought that was weird, given its specific to its implementation. What do y'all think?

I think ideally it'd be part of some persistentStorage revamp item

Reply:
I think it'd be okay to make a function in urlutils that takes in a componentManager instance and spits out the list of bad prefixes. We definitely don't want to duplicate this logic in multiple places. Actually, if it makes you feel better about the interface, your new urlutils method could just take in a function mapping component type to names. It could be totally agnostic about how that function is implemented or where it comes from. That would also make unit tests easier for it.